source/macos/AdaptiveCards/AdaptiveCards/Utils/HostConfigUtils.swift

```diff
@@ -39,7 +39,7 @@ extension ACSHostConfig {
 class FontUtils {
     private static var notFoundFontFamilies: Set<String> = []
     
-    static func getFont(for hostConfig: ACSHostConfig, with textProperties: ACSRichTextElementProperties) -> NSFont {
+    static func getFont(for hostConfig: ACSHostConfig, with textProperties: ACSRichTextElementProperties, attrsFont: NSFont? = nil) -> NSFont {
```

Comment:
keep descriptive names for variables.
prefer "attributesFont"  in place of "attrsFont"

Reply:
Yes, change it.